components/script/dom/webidls/Console.webidl

```diff
@@ -1,25 +1,27 @@
 /* This Source Code Form is subject to the terms of the Mozilla Public
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at https://mozilla.org/MPL/2.0/. */
-/*
- * References:
- *   MDN Docs - https://developer.mozilla.org/en-US/docs/Web/API/console
- *   Draft Spec - https://sideshowbarker.github.io/console-spec/
- *
- * © Copyright 2014 Mozilla Foundation.
- */
+
+// https://console.spec.whatwg.org/
```

Comment:
This needs to be added to https://dxr.mozilla.org/servo/source/python/tidy/servo_tidy/tidy.py#80